packages/vscode-extension-serverless-workflow-editor/src/extension/ServerlessWorkflowEditorChannelApiImpl.ts

```diff
@@ -188,4 +188,8 @@ export class ServerlessWorkflowEditorChannelApiImpl implements ServerlessWorkflo
   public async kogitoSwfLanguageService__getCodeLenses(args: { uri: string; content: string }): Promise<CodeLens[]> {
     return this.swfLanguageServiceChannelApiImpl.kogitoSwfLanguageService__getCodeLenses(args);
   }
+
+  public kogitoSwfLanguageService__moveCursorToNode(args: { nodeName: string }): void {
+    return this.swfLanguageServiceChannelApiImpl.kogitoSwfLanguageService__moveCursorToNode(args);
```

Comment:
Since this is void, you don't need to return anything here.
```suggestion
    this.swfLanguageServiceChannelApiImpl.kogitoSwfLanguageService__moveCursorToNode(args);
```